### ISSUE TITLE
3.0.5-beta Nuget Package not working

### ISSUE THREAD

**Issue by petedavis** [MassTransit/Automatonymous#19]
If you install with the latest prerelease I am only getting 3.0.4-beta, which is causing a whole heap of issues with trying to checkout MassTransit 3.0.11-beta as it as a dependency that cannot be resolved.

The 3.0.5 package is listed on nuget but that is not what is being installed when running a nuget install.

**Comment by petedavis** [comment]
@phatboyg there is a problem with the Automatonymous nuget package. 

This is what happens if you try and install the latest pre release SHOULD be 3.0.5-beta into an new blank project:
```
PM> Install-Package Automatonymous -Pre
Attempting to gather dependencies information for package 'Automatonymous.3.0.4-beta' with respect to project 'ClassLibrary1', targeting '.NETFramework,Version=v4.5'
Attempting to resolve dependencies for package 'Automatonymous.3.0.4-beta' with DependencyBehavior 'Lowest'
Resolving actions to install package 'Automatonymous.3.0.4-beta'
Resolved actions to install package 'Automatonymous.3.0.4-beta'
Adding package 'Automatonymous.3.0.4-beta' to folder 'c:\users\peter\documents\visual studio 2015\Projects\ClassLibrary1\packages'
Added package 'Automatonymous.3.0.4-beta' to folder 'c:\users\peter\documents\visual studio 2015\Projects\ClassLibrary1\packages'
Added package 'Automatonymous.3.0.4-beta' to 'packages.config'
Successfully installed 'Automatonymous 3.0.4-beta' to ClassLibrary1
```

**Comment by petedavis** [comment]
I have isolated this issue as a problem with the feed URL https://api.nuget.org/v3/index.json that is the default feed in VS 2015.

When you add a backup feed to the old v2 url 'https://www.nuget.org/api/v2/' to VS 2015 it now works.

```
PM> Install-Package Automatonymous -Pre
Attempting to gather dependencies information for package 'Automatonymous.3.0.5-beta' with respect to project 'ClassLibrary2', targeting '.NETFramework,Version=v4.5'
Attempting to resolve dependencies for package 'Automatonymous.3.0.5-beta' with DependencyBehavior 'Lowest'
Resolving actions to install package 'Automatonymous.3.0.5-beta'
Resolved actions to install package 'Automatonymous.3.0.5-beta'
Adding package 'Automatonymous.3.0.5-beta' to folder 'c:\users\peter\documents\visual studio 2015\Projects\ClassLibrary2\packages'
Added package 'Automatonymous.3.0.5-beta' to folder 'c:\users\peter\documents\visual studio 2015\Projects\ClassLibrary2\packages'
Added package 'Automatonymous.3.0.5-beta' to 'packages.config'
Successfully installed 'Automatonymous 3.0.5-beta' to ClassLibrary2
````

**Comment by petedavis** [comment]
This is been resolved by either NuGet/NuGetGallery#2683 or the new version release.
